app/controllers/api/teams.php

```diff
@@ -627,6 +627,10 @@
             throw new Exception('Team IDs don\'t match', 404);
         }
 
+        if ($membership->getAttribute('confirm') !== false) {
```

Comment:
```suggestion
        if ($membership->getAttribute('confirm') === true) {
```

Checking for acceptance feels a bit safer.